### DISCUSSION TITLE
[ARC 3.5+ (including 3.6.2)] On-Boot ARCadia-RS Broken

### DISCUSSION THREAD

**Discussion by HyperNuke632** [Raytwo/ARCropolis#405] [Issues] [answered]
System Firmware Version
14.1.2

Atmosphere Version
1.3.2

Smash Version
13.0.1

ARCropolis Version
3.6.2 (and every version after 3.5 iirc)

Can the crash be consistently reproduced?
Yes

At what moment does the crash happen?
When you go to ARCadia-RS to select and deselect mods, regardless if you change anything or not, if you leave the menu, the game never loads (softlocks on a black screen.)

Details:
I mean, just try it. It hasn't worked for me for a while. You can contact me on discord about it if you need proof, but as it stands it simply does not function.

**Comment by SorcererPonse** [comment]
Can confirm this to be an issue, I've had several times where I tried to pinpoint mods causing crashes and couldn't unload mods due to on-boot ARCadia-RS softlocking on black screen after closing the mod menu.

**Comment by Raytwo** [comment, marked as answer]
[Fixed](https://github.com/Raytwo/ARCropolis/issues/418)
